suse_migration_services/prechecks/pre_checks.py

```diff
@@ -32,9 +35,26 @@ def main():
     Checks whether
       - repositories' locations are not remote
       - filesystems in fstab are using LUKS encryption
+      - Multiple kernels are installed on the system and multiversion.kernels
+        in /etc/zypp/zypp.conf is not set to 'running, latest'
     """
     Logger.setup(system_root=False)
     log = logging.getLogger(Defaults.get_migration_log_name())
     log.info('Running pre migration checks')
+
+    cli_parser = argparse.ArgumentParser(
+        prog='suse-migration-pre-checks',
+        description='Check for existing conditions that will cause a migration to fail'
+    )
+
+    cli_parser.add_argument(
+        '-f',
+        '--fix',
```

Comment:
can this be `--fix-kernel` or `--fix-multikernels` ?
Something more specific, I would understand `fix` would fix _all_ checks instead of only the kernel

Reply:
We do not know what else will need to be fixed, if we make the option kernel specific now we will have an interface change later.

Reply:
marking resolved per Robert's comments